.travis.yml

```diff
@@ -47,15 +47,21 @@ jobs:
       env: TOXENV=py37
       before_install:
         - brew update
+        # update gcc first because it was breaking the py37 update as of 2018-10-23;
+        # might not be necessary in the future
+        - rm '/usr/local/include/c++'
```

Comment:
Following what `brew` recommends in its error message:

```
	Could not symlink include/c++
	Target /usr/local/include/c++
	already exists. You may want to remove it:
	  rm '/usr/local/include/c++'
```

@jaraco does this seem correct to you? Any other suggestion?

Reply:
Oh sorry, I'm mis-remembering, who contributed the macos builds was not @jaraco. 

@jirikuncar what do you think?

Reply:
Is this symlink in the Travis image already?  You might want to `ls -l` it in the beginning.

Reply:
This happens when also upgrading `numpy` (https://travis-ci.org/pytest-dev/pytest/jobs/444989259#L933), which maybe can be avoided with some option to "brew upgrade python" to not upgrade dependents?  (should be faster then, too)